README.md

```diff
@@ -300,80 +301,96 @@ A newly spawned agent won't contain any of the docker caches for the first run w
 
 ```yaml
 steps:
-  - label: ":docker: Build Intermediate Image"
-    plugins:
-      - docker-compose#v4.16.0:
-          build: myservice_intermediate  # docker-compose.yml is the same as myservice but has `target: intermediate`
-          image-name: buildkite-build-${BUILDKITE_BUILD_NUMBER}
-          image-repository: index.docker.io/myorg/myrepo/myservice_intermediate
-          cache-from:
-            - myservice_intermediate:index.docker.io/myorg/myrepo/myservice_intermediate:${BUILDKITE_BRANCH}
-            - myservice_intermediate:index.docker.io/myorg/myrepo/myservice_intermediate:latest
-  - wait
-  - label: ":docker: Build Final Image"
-    plugins:
-      - docker-compose#v4.16.0:
-          build: myservice
-          image-name: buildkite-build-${BUILDKITE_BUILD_NUMBER}
-          image-repository: index.docker.io/myorg/myrepo
-          cache-from:
-            - myservice:index.docker.io/myorg/myrepo/myservice_intermediate:buildkite-build-${BUILDKITE_BUILD_NUMBER}:intermediate  # built in step above
-            - myservice:index.docker.io/myorg/myrepo/myservice:${BUILDKITE_BRANCH}
-            - myservice:index.docker.io/myorg/myrepo/myservice:latest
-
+    - label: ":docker: Build Intermediate Image"
+      plugins:
+          - docker-compose#v4.16.0:
+                build: myservice_intermediate # docker-compose.yml is the same as myservice but has `target: intermediate`
+                image-name: buildkite-build-${BUILDKITE_BUILD_NUMBER}
+                image-repository: index.docker.io/myorg/myrepo/myservice_intermediate
+                cache-from:
+                    - myservice_intermediate:index.docker.io/myorg/myrepo/myservice_intermediate:${BUILDKITE_BRANCH}
+                    - myservice_intermediate:index.docker.io/myorg/myrepo/myservice_intermediate:latest
+    - wait
+    - label: ":docker: Build Final Image"
+      plugins:
+          - docker-compose#v4.16.0:
+                build: myservice
+                image-name: buildkite-build-${BUILDKITE_BUILD_NUMBER}
+                image-repository: index.docker.io/myorg/myrepo
+                cache-from:
+                    - myservice:index.docker.io/myorg/myrepo/myservice_intermediate:buildkite-build-${BUILDKITE_BUILD_NUMBER}:intermediate # built in step above
+                    - myservice:index.docker.io/myorg/myrepo/myservice:${BUILDKITE_BRANCH}
+                    - myservice:index.docker.io/myorg/myrepo/myservice:latest
 ```
 
 In the example above, the `myservice_intermediate:buildkite-build-${BUILDKITE_BUILD_NUMBER}` is one group named "intermediate", and `myservice:${BUILDKITE_BRANCH}` and `myservice:latest`
 are another (with a default name). The first successfully downloaded image in each group will be used as a cache.
 
+## Reusing previously built images
```

Comment:
This is the change in the readme that's not just formatting.

Reply:
FWIW, although for some reason GitHub is not showing it (?), I think that this branch drops the note that you added to the top of the README (I think for SOC 2 reasons?) in https://github.com/commonlit/docker-compose-buildkite-plugin/pull/1.

You can see this if you do `git diff origin/master..reintegrate-changes README.md`.

Reply:
Got it and pulled it back in. I think the diff is missing because we're changing history to get in line with the upstream repo.